tests/conftest.py

```diff
@@ -9,6 +9,7 @@
 import pandas as pd
 import pytest
 from fastapi.testclient import TestClient
+from flaky import flaky  # noqa  # pylint: disable=unused-import
```

Comment:
why is this needed?

Reply:
1. We could use `pytest.mark.flaky`, but it doesn't support args like `max_runs`
2. `# noqa  # pylint: disable=unused-import` is needed so flake8 and pylint won't error out.

Reply:
Are we going to though? Is there any other implementation for flaky? :)
I did it for long because it's shorter :)

Reply:
Yes, there are - couple are listed here https://docs.pytest.org/en/7.1.x/explanation/flaky.html#plugins
I dunno if we are going to change this TBH, but see no harm in keeping it like it is right now.